src/app/home/home.component.css

```diff
@@ -239,20 +270,8 @@
     background-color: #a9a9a9;
 }
 
-#offcanvasStartButton:hover {
```

Comment:
Think this code was duplicated, was also found in header styling, removed it here since `#offcanvasStartButton` is defined in the header, please double check this @Kledis98.

Reply:
Yes, it is doubled.